aws/biganimal-csp-setup

```diff
@@ -239,7 +239,7 @@ create_iam_policy()
 EOF
   )
   echo "$biganimal_policy" > "${TMPDIR}"/biganimal_policy.json
```

Comment:
if we have `aws/biganimal_AWS_basepolicy.json` already in the place, why not just 
```
cp aws/biganimal_AWS_basepolicy.json "${TMPDIR}"/biganimal_policy.json
```

Reply:
unfortunately right now the `biganimal` CLI which we use to automate the execution of this script (to make it more easy to use from a customer perspective) is not downloading any other file in addition to the bash script itself, so we can't do that, an option here can be something similar https://github.com/EnterpriseDB/cloud-utilities/blob/main/azure/biganimal-csp-setup#L188-L200 but I'd prefer to discuss/make this change in another PR, WDYT?